source/app/shared/utilities.ts

```diff
@@ -23,7 +23,24 @@ function formatRoute(original: string, ...args: string[]): string {
 }
 
 function createPlayFabLink(titleId: string, uri: string, isReact: boolean): string {
-    return `https://developer.playfab.com/en-US/${isReact ? `r/t/` : ``}${titleId}/${uri}`;
+    const playFabMainProdUrl = ".playfabapi.com";
+
+    var urlRoot;
+
+    if(((PlayFab as any)._internalSettings.productionServerUrl === playFabMainProdUrl))
```

Comment:
Since we're hacking against the SDK's internal settings, I'd like to factor this out a little bit. Could you make new functions here in `utilities.ts` to get and set the `productionServerUrl`?

e.g.

``` typescript
function setPrivateCloud(cloud: string): void {
    (PlayFab as any)._internalSettings.productionServerUrl = `.${cloud}.playfabapi.com`;
}

function getPrivateCloud(): string {
    return (PlayFab as any)._internalSettings.productionServerUrl;
}
```

Reply:
getPrivateCloud is basically a hack since we don't have it in the URI. I am going to leave it as is and will fix in a follow up PR.

---
In reply to: [310237167](https://github.com/jordan-playfab/playfab-liveops-game/pull/22#discussion_r310237167) [](ancestors = 310237167)